components/elements/AccountDropdown.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import styled from 'styled-components';
+import useUser from '../auth/User';
+
+const AccountDropdown = ({ children }) => {
+  const user = useUser();
+
+  let userInitial;
+  if (user && user.name) {
+    userInitial = user.name[0].toUpperCase();
+  }
+  return (
+    <>
+      <Circle>
+        <div className="initial">{userInitial}</div>
+        <div className="menu">{children}</div>
+      </Circle>
+    </>
+  );
+};
+
+const Circle = styled.div`
+  width: 50px;
+  height: 50px;
+  border-radius: 50%;
+  position: relative;
+  background-color: var(--Primary);
+  z-index: 1000;
+  .initial {
+    font-size: 3rem;
+    color: var(--OnMidground);
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+  }
+  &:hover .menu {
+    display: block;
+  }
+  .menu {
+    width: 200px;
+    position: absolute;
+    top: 100%;
+    right: 0;
+    padding: 1rem;
+    display: none;
+    background-color: white;
+    border: 1px solid var(--Primary);
+    a,
+    button {
+      line-height: 1em;
+      width: 100%;
+      display: block;
+      padding: 0.75rem 0.5rem;
+      text-align: right;
+      &:hover,
+      &:focus {
+        background-color: var(--Accent);
+        text-decoration: none;
+      }
+      &:hover::after,
+      &:focus::after {
```

Comment:
`focus` doesn't  have a text-decoration. why do you need it?

Reply:
And if you still do want to combine it, you can still use the `scss` approach by nesting:

```
&:hover,
&:focus {
    background-color: var(--Accent);
     text-decoration: none;
      &:after {
          content: '';
           //etc
       }
}
```

Reply:
> `focus` doesn't  have a text-decoration. why do you need it

Oh, I was looking at mdn and I thought I understood that on focus links get at outline. You can remove it so long as you include another focus state?

Reply:
Yes, you're right. Is gets an outline, but not a text-decoration.`outline` is a css property you can use.
```css
&:focus {
  outline: 1px dashed red;
}
```

BUT, I want to stress, from a UX perspective, to be careful changing this. As the user can recognise it's focus point with this. For example when you navigate the site with only your keyboard

Reply:
Yes, I'm also learning about disable vs aria disable... there is so much to learn! It must suck so hard to have accessibility issues, like, no tech is truly accessible. Oof. I'm so glad I have eye sight.